touchforms/backend/touchcare.py

```diff
@@ -31,6 +36,171 @@
 logger = logging.getLogger('formplayer.touchcare')
 
 
+def get_restore_url(criteria=None):
+    query_url = '%s?%s' % (settings.RESTORE_URL, urllib.urlencode(criteria))
+    return query_url
+
+
+class CCInstances(InstanceInitializationFactory):
+
+    def __init__(self, sessionvars, auth, restore=None, force_sync=False, form_context=None):
+        self.vars = sessionvars
+        self.auth = auth
+
+        if settings.USES_SQLITE:
+            self.username = sessionvars['username']
+            self.sandbox = SqlSandboxUtils.getStaticStorage(self.username)
+            self.host = sessionvars['host']
+            self.domain = sessionvars['domain']
+            self.query_func = query_factory(self.host, self.domain, self.auth, 'raw')
+            self.query_url = get_restore_url({'as': self.username + '@' + self.domain, 'version': '2.0'})
+
+            if force_sync or self.needs_sync():
+                self.perform_ota_restore(restore)
+        else:
+            self.fixtures = {}
+            self.form_context = form_context or {}
+
+    def clear_tables(self):
+        db_name = self.username + ".db"
+        os.remove(db_name)
+        self.sandbox = SqlSandboxUtils.getStaticStorage(self.username)
+
+    def perform_ota_restore(self, restore=None):
+        self.clear_tables()
+        if not restore:
+            restore_xml = self.get_restore_xml()
+            ParseUtils.parseXMLIntoSandbox(restore_xml, self.sandbox)
+        else:
+            restore_file = restore
+            ParseUtils.parseFileIntoSandbox(File(restore_file), self.sandbox)
+        persistence.postgres_set_sqlite(self.username, 1)
```

Comment:
We're not doing app restores yet so this isn't being used (there is no app version), put the field in for future iterations, but maybe should just remove?